queue/README.md

```diff
@@ -0,0 +1,117 @@
+
+# VALIDATOR QUEUE
+
+## Introduction
+
+The queue objective is to receive messages and pop the right msg by to specify priority.
+The priority is based on what will achieve the quickest consensus and keep the node in sync with other peers.
+
+### Queue
+
+Receive msg and add to stack. queue provide interface with ADD, POP, DELETE functions. 
+
+### Index Points
+
+- Height
+- Slot
+- Consensus
+  - Propose
+  - Prepare
+  - Commit
+  - Commit (decided)
+  - Change Round
+- Post Consensus
+  - PostConsensusPartialSig
+  - RandaoPartialSig
+  - SelectionProofPartialSig
+  - ContributionProofs
+
+### Priority
+
+#### V1
+
+1. Higher height (all above the current height)
+2. If No Instance Running
```

Comment:
no running instance means also decided?

Reply:
Let's revisit the ordering after SSV is adapted to use it and we can test it for real (working on it now.)